### PR TITLE
[5.2] Add legacy message to content modules

### PR DESCRIPTION
**Pull request by bembelimen**
### Summary of Changes
With #43738 we have a "one for all" module for certain content modules. This PR adds an info that users should use the new module instead.

### Testing Instructions
- Appy PR

### Actual result BEFORE applying this Pull Request
No messages

### Expected result AFTER applying this Pull Request
- Content modules have an info text
grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by fgsw** [comment]
I have tested this item :white_check_mark: successfully on 0a544753350f1ee21d56b8f59937e843bc988ef9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44366">issues.joomla.org/tracker/joomla-cms/44366</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 0a544753350f1ee21d56b8f59937e843bc988ef9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44366">issues.joomla.org/tracker/joomla-cms/44366</a>.</sub>

File: language/en-GB/mod_articles_popular.sys.ini [hunk 1]
```diff
@@ -3,6 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-MOD_ARTICLES_POPULAR="Articles - Most Read"
+MOD_ARTICLES_POPULAR="Articles - Most Read (deprecatd)"
```
Comment (Quy) [inline language/en-GB/mod_articles_popular.sys.ini]:
```suggestion
MOD_ARTICLES_POPULAR="Articles - Most Read (deprecated)"
```

File: language/en-GB/mod_articles_category.ini [hunk 2]
```diff
@@ -72,4 +72,4 @@ MOD_ARTICLES_CATEGORY_READ_MORE="Read more: "
 MOD_ARTICLES_CATEGORY_READ_MORE_TITLE="Read More &hellip;"
 MOD_ARTICLES_CATEGORY_REGISTER_TO_READ_MORE="Register to read more"
 MOD_ARTICLES_CATEGORY_UNTAGGED="Untagged"
-MOD_ARTICLES_CATEGORY_XML_DESCRIPTION="This module displays a list of articles from one or more categories."
+MOD_ARTICLES_CATEGORY_XML_DESCRIPTION="This module displays a list of articles from one or more categories. This module is deprecated, use the \"Articles\" module from now on."
```
Comment (Quy) [inline language/en-GB/mod_articles_category.ini:75]:
```suggestion
MOD_ARTICLES_CATEGORY_XML_DESCRIPTION="This module displays a list of articles from one or more categories. This module is deprecated, use the \"Articles\" module instead."
```

Comment (Quy) [inline language/en-GB/mod_articles_category.ini]:
```suggestion
MOD_ARTICLES_CATEGORY_XML_DESCRIPTION="This module displays a list of articles from one or more categories. This module is deprecated, use the \"Articles\" instead."
```

**Comment by brianteeman** [comment]
I am not in favour of such a public deprecation notice. 

1. the reality is that these modules are here to stay unless someone works out a way to migrate existing instances of those modules to the new module.

2. the meaning of the word "deprecated" to be "superseded" is not common outside of the programming world. using it here is meaningless and creates confusion if it is not understood. 

3. finally this breaks even the programmers usage of the word "deprecated" as it fails to say when it will be "removed" and the language is not clear what you should be using instead.

This is a change for changes sake and benefits no one.

**Comment by brianteeman** [comment]
Maybe it's an English language issue and instead of writing deprecated it would be better to write legacy

**Comment by laoneo** [comment]
And it should go into 5.3.

File: language/en-GB/mod_articles_popular.sys.ini [hunk 3]
```diff
@@ -3,6 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-MOD_ARTICLES_POPULAR="Articles - Most Read"
+MOD_ARTICLES_POPULAR="Articles - Most Read (deprecated)"
```
Comment (Quy) [inline language/en-GB/mod_articles_popular.sys.ini]:
```suggestion
MOD_ARTICLES_POPULAR="Articles - Most Read ((Legacy)"
```

File: language/en-GB/mod_articles_popular.sys.ini [hunk 4]
```diff
@@ -3,6 +3,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-MOD_ARTICLES_POPULAR="Articles - Most Read"
+MOD_ARTICLES_POPULAR="Articles - Most Read ((Legacy)"
```
Comment (Quy) [inline language/en-GB/mod_articles_popular.sys.ini]:
```suggestion
MOD_ARTICLES_POPULAR="Articles - Most Read (Legacy)"
```

**Comment by Quy** [comment]
Missing `Articles - Archived` per the following:
```
Articles - Archived -> Filter Archived Articles only
Articles - Category -> Select category, play with parameter, use filter, group them, etc.
Articles - Latest -> Change ordering to newest first
Articles - Most Read -> Change ordering to hits
Articles - Newsflash -> Select category, play with parameter, use filter, group them, etc.
```

**Comment by Kostelano** [comment]
> Missing Articles - Archived per the following:

@Quy 
Does the new module have a layout for displaying a list of months, as is implemented in the archive module?

For now, the new module can group and display archive articles, but not a list of months (correct me if I'm wrong and missed this).

**Comment by Quy** [comment]
> > Missing Articles - Archived per the following:
> 
> @Quy Does the new module have a layout for displaying a list of months, as is implemented in the archive module?
> 
> For now, the new module can group and display archive articles, but not a list of months (correct me if I'm wrong and missed this).

Sorry I have not used the Articles module yet to answer your question. Based on PR #43738 description, this is one of the modules being replaced.

**Comment by softforge** [comment]
> Maybe it's an English language issue and instead of writing deprecated it would be better to write legacy

Deprecated and legacy convey two distinctly different meanings.

Deprecated means the "thing" (function, API, feature) is still in the code and available but will be removed in future versions and better alternatives exist, that continued use will at some point cause problems.

Legacy means the "thing" is still in active use, works but may be outdated, needs to be maintained for compatibility, isn't necessarily bad or discouraged, and may be difficult to replace.

So in simple terms
Deprecated = "Stop using this it's going away"
Legacy= "This is old but still valid and supported"

If you want to make usage clear then say (Removed in future releases)

This conveys the need to not start using it. Whereas legacy does not convey the discouragement needed.

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.3-dev.

**Comment by exlemor** [comment]
I have tested this item :white_check_mark: successfully on ecd6eae2bd770571eef8b02a1f65b3ae1e18bd47<br /><br />I have tested this successfully. 


(Legacy is a good choice, it makes me think, I probably should start looking/using the new version soon with the idea it is better in x ways).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44366">issues.joomla.org/tracker/joomla-cms/44366</a>.</sub>

**Comment by degobbis** [comment]
I have tested this item :white_check_mark: successfully on ecd6eae2bd770571eef8b02a1f65b3ae1e18bd47<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44366">issues.joomla.org/tracker/joomla-cms/44366</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44366">issues.joomla.org/tracker/joomla-cms/44366</a>.</sub>

**Comment by rakibhossainraju** [comment]
I have tested this item :white_check_mark: successfully on ecd6eae2bd770571eef8b02a1f65b3ae1e18bd47<br /><br />I have successfully tested it.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44366">issues.joomla.org/tracker/joomla-cms/44366</a>.</sub>

**Comment by HLeithner** [comment]
thanks
